src/open_inwoner/configurations/bootstrap/constants.py

```diff
@@ -0,0 +1,229 @@
+from mozilla_django_oidc_db.models import UserInformationClaimsSources
+from zgw_consumers.api_models.constants import VertrouwelijkheidsAanduidingen
+
+from open_inwoner.openzaak.models import generate_default_file_extensions
+
+from .utils import generate_api_fields_from_template
+
+CONFIDENTIALITY_CHOICES = [
+    choice[0] for choice in VertrouwelijkheidsAanduidingen.choices
+]
+FILE_EXTENSION_CHOICES = [ext for ext in generate_default_file_extensions()]
+USER_INFO_CLAIM_SOURCES_CHOICES = [
+    choice[0] for choice in UserInformationClaimsSources.choices
+]
+
+
+siteconfig_fields = {
+    "name": {"values": "string"},
```

Comment:
I'm slightly mystified why this is here: went for pulling the fields and the meta information from the config model, instead of manually declaring every field using the class+descriptors because you didn't want to make declarations for every field. 

But this still declares info for every field.. and as loose dictionaries instead of something more controlled like with the class+descriptors. :confused: